### ISSUE TITLE
Feature: Add command line option to use stdin or text instead of file as a source

### ISSUE THREAD

**Issue by FrancisMurillo** [miso-belica/sumy#66]
Nice library.

Using a temporary file is a little hastle but it would be nice if you can just plug it in as a command line option or pipe it.

**Comment by miso-belica** [comment]
Hi, thanks. Sumy provides the possibility to read from stdin. Just omit URL/File parameter and that's it :) Let me know if it's ok for you, or need sth else.

**Comment by FrancisMurillo** [comment]
Doh! 

Would it be pushing it if an option is what I really needed? I use this with `Emacs` and `start-process`, so if I pipe over it I still have to create a temporary buffer to hold the result which is really not that different if I stick with a temporary file.

**Comment by miso-belica** [comment]
Sorry, I don't think I understand. I don't know Emacs but IMHO you can stream text into sumy like `./stream_text | sumy lsa`. Can you provide CLI interface that you expect from me in [docopt format](http://docopt.org/)?

**Comment by FrancisMurillo** [comment]
Emacs was just an example. If Python is a good example using the `sh` library, it would look something similar in `Emacs`

````
from sumy import sh

raw_text = 'Blah blah blah I like cats blah blah blah'

sumy('lex-rank', '--length', '5', '--text', my_raw_text)
```

The `docopt` format I think would be.

```
sumy (luhn | edmundson | lsa | text-rank | lex-rank | sum-basic | kl) [--length=<length>] [--language=<lang>] [--stopwords=<file_path>] [--format=<format>] --text=<text>
```
Maybe something like that. My actual call would then be

```
(setq my-raw-text "I love puspins")
(call-process "sumy" "lex-rank" "--length" "5" "--text" my-raw-text)
```

**Comment by miso-belica** [comment]
This was fixed in c382a42, thanks and nice day :)
